File: SDKs/sportsdata.io/soccer-v3-stats/1.0/go/pkg/models/operations/venues.go
package operations

type VenuesFormatEnum string

const (
	VenuesFormatEnumXML  VenuesFormatEnum = "xml"
	VenuesFormatEnumJSON VenuesFormatEnum = "json"
)

type VenuesPathParams struct {
	Format VenuesFormatEnum `pathParam:"style=simple,explode=false,name=format"`
}

type VenuesRequest struct {
	PathParams VenuesPathParams
}

type VenuesResponse struct {
	ContentType string
	StatusCode  int64
	Venues      []interface{}
}

File: SDKs/sportsdata.io/csgo-v3-scores/1.0/typescript/src/sdk/models/operations/venues.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";


export enum VenuesFormatEnum {
    Xml = "xml",
    Json = "json"
}


export class VenuesPathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=format" })
  format: VenuesFormatEnum;
}


export class VenuesRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  pathParams: VenuesPathParams;
}


export class VenuesResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;

  @SpeakeasyMetadata()
  venues?: any[];
}
